src/settings/BatchGroupConfigurationSettings/BatchVouchers/BatchVouchers.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { IconButton } from '@folio/stripes/components';
+
+const BatchVouchers = ({ downloadBatchVouchers }) => (
+  <IconButton
```

Comment:
so, you component is just an usage of `<IconButton />`? IMHO, it looks like over-engineering.